Modified to resolve issue with conflicting  provider_paths

remoorejr opened the pull request.

Commit:
Modified to resolve issue with conflicting  provider_paths

disusered commented:
Thank you for this. I am swamped at work but will give it priority this weekend. For context https://github.com/disusered/cordova-open/issues/64#issuecomment-494887456

marcomilone commented:
I'm using it also,
with same problem.

Thank you very much for your work.

Commit:
Update plugin.xml

Added ANDROID_SUPPORT_V4_VERSION to avoid errors with android.support.v4.content

remoorejr committed:
Modified to use AndroidX library, bump version

remoorejr committed:
Update README.md

remoorejr committed:
Update plugin.xml

Load dependency from github repo

remoorejr committed:
Update plugin.xml

add dependency versions
